src/Test/FixturesTrait.php

```diff
@@ -60,6 +61,25 @@ public function loadFixtureFiles(array $paths = [], bool $append = false, ?strin
         return $dbTool->loadAliceFixture($paths, $append);
     }
 
+    /**
+     * This loads all the fixtures defined in the project, including ordering
+     * them, e.g. by the DependentFixtureInterface. The call to this method
+     * does the same as running the console command doctrine:fixtures:load,
+     * including the use of the group parameter.
+     */
+    protected function loadAllFixtures(array $groups = []): ?AbstractExecutor
```

Comment:
That method will have to go there: https://github.com/liip/LiipTestFixturesBundle/blob/2.x/src/Services/DatabaseTools/AbstractDatabaseTool.php